website/pages/style.css

```diff
@@ -56,3 +56,7 @@
   border-radius: 3px;
   box-shadow: 0 14.3254px 14.3254px rgba(37, 41, 55, 0.16);
 }
+
+.boundary-clickthrough-desktop-video {
+  max-width: 800px;
```

Comment:
Why set the desktop video to a different size?

Reply:
Due to the full width capture of desktop, the inputs weren't readable at existing 500px width. Is there a better way to address this?